homeassistant/components/sia/hub.py

```diff
@@ -110,14 +110,12 @@ def update_accounts(self):
             self.sia_client.accounts = self.sia_accounts
             return
         # the new client class method creates a subclass based on protocol, hence the type ignore
-        self.sia_client = (
-            SIAClient(  # pylint: disable=abstract-class-instantiated # type: ignore
```

Comment:
mypy started flagging the type ignore as unnecessary after removing the pylint one; I guess the syntax didn't work that way, and when it started to work, the ignore was found unneeded.